### PR TITLE
Create aggland, njeru, njmi, and j:lexnp and upgrade lexington

### PR DESCRIPTION
**Pull request by Quivico**

### CODE REVIEW

File: data/CompliantSwitches/njeru.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+description = "New Jerusalem."
```
Comment (Ameliorate) [inline data/CompliantSwitches/njeru.toml:1]: Can you add `aliases`, like in https://github.com/Ameliorate/KANI/blob/master/data/CompliantSwitches/0,0.toml#L1 or https://github.com/Ameliorate/KANI/blob/master/data/CompliantSwitches/coe.toml#L1 ?

Comment (Quivico) [reply data/CompliantSwitches/njeru.toml:3]: Aliases added.

File: data/TerminalDestinations/np.toml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+aliases = ['north pole']
```
Comment (Ameliorate) [inline data/TerminalDestinations/np.toml:1]: Can you add an alias for `lexington vault museum`, like `aliases = ['north pole', 'lexington vault museum']

Comment (Quivico) [reply data/TerminalDestinations/np.toml]: Added.

**Comment by Ameliorate** [comment]
I'm not sure if you wanted it merged now, but I'll merge it anyways because it works and there are some destinations that are probably broken without this PR.

**Pull request merged**